sirepo/const.py

```diff
@@ -4,9 +4,23 @@
 :copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
-from __future__ import absolute_import, division, print_function
-
+from pykern.pkcollections import PKDict
```

Comment:
Do not import at this level. Put a comment to that effect.

Reply:
I'm not sure what "this level" means? No imports in const.py? Does that mean `PORT_DEFAULTS` has to live in a function, or be a regular dict, or broken up into individual constants?